### PR TITLE
create trackHub directories for the BED sets

### PR DESCRIPTION
**Pull request by xuebingjie1990**
Currently the trackHub directories are pre--computed during the bedbuncher step.   In the future, we may want to make them served by API endpoints so that on-the-fly bedsets can be exported as track hubs.

### CODE REVIEW

**Comment by stolarczyk** [comment]
Merge this right away if you're ready and then we'll open a PR to master and review.

**Comment by nsheff** [comment]
> Currently the trackHub directories are pre--computed during the bedbuncher step. In the future, we may want to make them served by API endpoints so that on-the-fly bedsets can be exported as track hubs.

This kind of stuff makes a good issue, so we can keep track of the ideas. in a description to a PR it will get forgotten once the PR is merged.

File: README.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
 Pipeline designed to create **bedsets** (sets of BED files) that will be retrieved from bedbase.
 
 Examples of bedsets:
-- BED files with regions count greater than 1000 (query: `regions_no>1000`)
+- BED files with regions count greater than 1000 (query: `regions_no>%s` query_val: `1000`)
 - BED files produced in ChiP-seq experiments
```
Comment (nsheff) [inline README.md]: it's ChIP-seq (note capitalization)

**Comment by nsheff** [comment]
track hubs appears to not be working from the bedset splash page link...

http://data.bedbase.org/outputs/bedbuncher_output/a796b6fd6533deec19b4ed01e63aaf39/bedsetHub/hub.txt

image

**Comment by nsheff** [comment]
(I'm looking at this bedset: http://dev1.bedbase.org/bedsetsplash/a796b6fd6533deec19b4ed01e63aaf39)

Maybe unrelated to the PR but it caused me to start looking at bedset trackhubs.

**Comment by xuebingjie1990** [comment]
> (I'm looking at this bedset: http://dev1.bedbase.org/bedsetsplash/a796b6fd6533deec19b4ed01e63aaf39)


I haven't upload the bedbuncher data to s3 yet. Because there are 3 jobs that are still running.

**Pull request merged**